packages/console-types/src/types/catalog/custom-resource-definitions/application.crd.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2025 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { catalogApplication } from '../well-known-items'
+import { PublicCatalogCRD } from '.'
+
+const applicationCRD: PublicCatalogCRD = {
+  name: 'application',
+  itemId: 'application-definition',
+  description: 'Application Custom Resource Definition',
+  type: 'custom-resource-definition',
+  tenantId: 'mia-platform',
+  isVersioningSupported: false,
+  visibility: { public: true },
+  isLatest: true,
+  resources: {
+    name: 'application',
+    validation: {
+      jsonSchema: {
+        ...catalogApplication.resourcesSchema,
+        default: {
+          services: {
+            'change-with-your-plugin-name': {
```

Comment:
why is there a placeholder definition here? 🤔

Reply:
We are using the `default` field of CRDs as placeholders for the Software Catalog (you can find them also in the CVU)

I suppose we could use the `examples` field, but it might be a bit less intuitive since it's an array and there may be some uncertainty on which item to consider